sql/updates/world/master/2021_99_99_99_world_monk_stagger.sql

```diff
@@ -0,0 +1,9 @@
+-- Stagger
+
```

Comment:
```suggestion
```

Reply:
I like this empty line :p do I have to remove it?

Reply:
Cosmetic, not very important. We're just used to seeing the first comment line directly on top of the following code, at least when there is a complete `DELETE FROM / INSERT INTO` query.

Reply:
mah you bought me with "used to seeing" and "codestyle" - I can't resist a standard.
removed the empty line.